cmd/apiserver/main.go

```diff
@@ -24,6 +24,10 @@ import (
 	"os"
 	"strconv"
 
+	"github.com/google/exposure-notifications-server/pkg/keys"
+	"github.com/google/exposure-notifications-server/pkg/logging"
+	"github.com/google/exposure-notifications-server/pkg/observability"
+	"github.com/google/exposure-notifications-server/pkg/server"
```

Comment:
add these to the second package block... altho I don't think you need them?

Reply:
I think those are needed. I just sorted the imports already there. Refactored to proper import block and then sorted the imports